flipt-client-go/evaluation.go

```diff
@@ -157,11 +163,15 @@ func (e *Client) EvaluateBoolean(_ context.Context, flagKey, entityID string, ev
 		return nil, err
 	}
 
-	return br, nil
+	if br.Status == statusSuccess {
```

Comment:
we could make this a non breaking API change by returning `br` here and still check if `br.Status == success` and return an error if not right?

i just worry about making this breaking change in only one sdk until we have tried it in all client side sdks to make sure it is the 'shape' of the responses that we want.

wdyt?

Reply:
yes, we can do that.  Please revert those or I will do it tomorrow.